pkg/install/install.go

```diff
@@ -196,6 +197,7 @@ func (i *Installer) Install(ctx context.Context, installConfig *installconfig.In
 			condition{i.operatorConsoleReady, 10 * time.Minute},
 			condition{i.clusterVersionReady, 30 * time.Minute},
 			action(i.disableAlertManagerWarning),
+			action(i.fixCloudConfig), // TODO(mjudeikis): Remove me when pvc fix lands
```

Comment:
not as simple as 'Remove me when', right?

Reply:
Feature 7484515

Reply:
Should we add a reference to the work item into the comment?